rest-core/src/Rest/Error.hs

```diff
@@ -46,11 +45,11 @@ domainReason = CustomReason . DomainReason
 infixl 3 >|<
 -- | Combine two ExceptT computations yielding the last error if both fail.
 -- This prevents the need for a Semigroup or Monoid instance for the error type, which is necessary if using (<!>) or (<|>) respectively.
-(>|<) :: (Applicative m, Monad m) => ExceptT e m a -> ExceptT e m a -> ExceptT e m a
-a >|< b = mapE getLast (mapE Last a <!> mapE Last b)
-  where
```

Comment:
Ah yes, we wanted a `Semigroup` for `Last`, otherwise if we got a `Monoid` we'd be stuck with a `Maybe e`.